src/components/Loader.scss

```diff
@@ -0,0 +1,248 @@
+//------------------------------------------------------------//
+
+// * Loaders
+// * 1 Loading Animations
+// * 1.a Jello Animation
+// * 1.b Pulse Animation
+// * 1.c Spin Animation
+// * 2 Page Loader
+// * 2.a Page Loader Modifiers
+// * 3 Block Loader
+
+// Notes:
+// * Animations are indepented from Loaders to provide more
+// flexibility.
+
+// TODO : Add page loader midifiers for full screen and such
+// TODO : Add Indicators and Loader variable settings
+// TODO : Add styles for Element Loaders (if needed)
+// TODO : cleanup PageLoader--centered to be more flexible
+
+//------------------------------------------------------------//
+
+// * 1. Loading Indicators
+
+// * 1.a Jello Indicator
+
+.JelloAnimation {
+  display: inline-block;
+  position: relative;
+  height: scaleGrid(5);
+}
+
+.JelloAnimation-box {
+  width: scaleGrid(4);
+  height: scaleGrid(4);
+  background: $ts-orange;
+  animation: jelloBox 500ms linear infinite;
+  position: absolute;
+  top: 0;
+  left: 0;
+  border-radius: $border-radius-small;
+}
+
+.JelloAnimation-shadow {
+  width: scaleGrid(4);
+  height: $su-xsmall;
+  background: $cu-divider--light;
+  position: absolute;
+  bottom: negative($su-xsmall);
+  left: 0;
+  border-radius: 50%;
+  -webkit-animation: jelloShadow 500ms linear infinite;
+  animation: jelloShadow 500ms linear infinite;
+}
+
+@mixin jelloBox($prefix) {
+  17% {
+    border-bottom-right-radius: $border-radius-small;
+  }
+  25% {
+    #{$prefix}transform: translateY($su-small) rotate(22.5deg);
+  }
+  50% {
+    border-bottom-right-radius: $su-xlarge;
+    #{$prefix}transform: translateY($su-default) scale(1,.9) rotate(45deg) ;
+  }
+  75% {
+    #{$prefix}transform: translateY($su-small) rotate(67.5deg);
+  }
+  100% {
+    #{$prefix}transform: translateY(0) rotate(90deg);
+  }
+}
+
+@keyframes jelloBox {
+  @include jelloBox('');
+}
+
+@-webkit-keyframes jelloBox {
```

Comment:
I _think_ we are using Autoprefixer? We shouldn't need to add prefixes within the CSS.